cli/native/src/main/scala/com/lightbend/rp/reactivecli/process/NativeProcess.scala

```diff
@@ -37,11 +37,14 @@ object NativeProcess extends LazyLogging {
    * the command will be processed by `sh` per POSIX specification.
    *
    * This means that we can simply enclose each argument in a single quote. However, if a single quote occurs in
-   * an argument, we special case that by enclosing it in double quotes.
+   * an argument, we special case that by enclosing it in double quotes. Also, we don't enclose "|" pipes and
+   * stdin/stdout redirects.
    */
   private[NativeProcess] def command(args: Seq[String]): String = {
-    def escape(s: String): String =
-      "'" + s.replaceAllLiterally("'", "'\"'\"'") + "'"
+    def escape(s: String): String = {
+      if (s == "|" || s == "<" || s == ">") s
+      else "'" + s.replaceAllLiterally("'", "'\"'\"'") + "'"
+    }
```

Comment:
This feels a bit 'leaky' to me -- the intent is to remove any special interpretation as it may be supplied by user input.

One alternative idea is that we could at make this more explicit? Add an argument controlling this behavior (perhaps `def command(args: Seq[String], passthrough: Set[String] = Set.empty`), and thread it through in the package class as well? Then from the call site where you need usage of `|` `<` `>` you can just be explicit about it, e.g. `cmd(Seq("hello", "world", ">", "/dev/null"), Seq(">"))`

Also, what about the JS build? This wouldn't work there would it?

Reply:
Good point, I was writing to a temp file previously and that worked with JS but piping will not. I'll boot up Windows VM again and address this.